### PR TITLE
Carousel component for the main screen

### PR DESCRIPTION
**Pull request by hueitan**
🚧 **not ready yet** 🚧 

Phabricator Link : https://phabricator.wikimedia.org/T241066

To do

- [x] Video Component
- [x] potd api
- [x] motd api
- [x] Carousel Component
- [ ] Refactoring
- [ ] Styling
- [ ] i18n
- [ ] watch videos embedded within an article

### Note

| Video Component | Carousel 1 | Carousel 2 |
| --- | --- | --- |
| Capture | Capture | Capture2|

### CODE REVIEW

File: src/hooks/usePotd.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'preact/hooks'
+
+export const usePotd = () => {
```
Comment (medied) [inline src/hooks/usePotd.js]: Does "potd" stand for photo of the day?

Comment (hueitan) [reply src/hooks/usePotd.js]: Pictures

Comment (jpita) [reply src/hooks/usePotd.js]: I'm all for full name variables, we don't pay per character written

**Comment by hueitan** [comment]
To follow up, please check the latest thread in https://phabricator.wikimedia.org/T241066

**Pull request closed without merging**